app/src/main/res/values-iw/strings.xml

```diff
@@ -78,4 +78,12 @@
   <string name="message_notification_title">הודעה חדשה</string>
   <string name="pref_general_summary_template">גרסא %1$s</string>
   <string name="pref_general_about_title">אודות</string>
+  <string name="menu_action_free_draw_title">ציור חופשי</string>
+  <string name="free_draw_title">ציור חופשי</string>
+  <string name="pref_notifications_vibrate_title">רטט</string>
+  <string name="pref_notifications_ringtone_title">רינגטון</string>
+  <string name="pref_notifications_ringtone_silent_summary">שקט</string>
+  <string name="pref_notifications_header_title">התראות</string>
+  <string name="toast_two_fingers_text">הזז את המפה באמצעות שתי אצבעות</string>
+  <string name="pref_notifications_alerts_only_title">הצג התראות דחופות בלבד</string>
 </resources>
```

Comment:
Reorganize this file. You have some weird things going on here... Duplicates etc.